tests/bias/test_presrat_bias_correction.py

```diff
@@ -0,0 +1,842 @@
+"""Validating PresRat correction procedures
+
+
+Relevant resources used in the tests:
+- FP_CC: Filename of standard biased dataset.
+- fut_cc: Future dataset sample based on FP_CC + an offset + small noise
+- fp_fut_cc: Filname to `fut_cc`.
+- fut_cc_notrend: Future dataset identical to FP_CC, i.e. no trend.
+- fp_fut_cc_notrend: Filename to fut_cc_notrend.
+- presrat_params: Parameters of reference to test PresRat (using fp_fut_cc).
+- presrat_notrend_params: Quantiles of future (mf) are identical to bias
+  reference (mh). Thus, there is no trend in the model.
+- presrat_identity_params: All distributions (oh & mf) are identical to mh,
+    i.e. observations equal to model that doesn't change on time.
+- presrat_nochanges_params: Like presrat_identity_params, but also all
+    zero_rate are zeros, i.e. no values should be forced to be zero.
+- presrat_nozeros_params: Same of presrat_params, but no zero_rate, i.e.
+    all zero_rate values are equal to 0 (percent).
+"""
```

Comment:
I think we need to add a test that looks at the fundamental benefits of presrat: 1) preserve the model predicted change in fraction of nonprecipitating days (with the exception of models with strong dry bias) 2) preservation of mean precip change. We can do this with the `fp_fut_cc` file constructors if they're ready or make dummy distributions for single pixels/months in the test.

Reply:
On my notes, this is the last requirement.